.expo/packager-info.json

```diff
@@ -1,6 +1,10 @@
 {
   "devToolsPort": 19002,
-  "expoServerPort": 19000,
-  "packagerPort": 19001,
-  "packagerPid": 16948
+  "expoServerPort": null,
+  "packagerPort": null,
+  "packagerPid": null,
+  "expoServerNgrokUrl": null,
+  "packagerNgrokUrl": null,
+  "ngrokPid": null,
+  "webpackServerPort": null
```

Comment:
the ports all null can run expo start or not? Cos usually expo start on port 19001 etc

Reply:
when I run the port number automatically gets filled up on my end